corehq/apps/userreports/tests/test_data_source_partitioning.py

```diff
@@ -14,7 +14,7 @@
 from corehq.pillows.case import get_case_pillow
 
 
-EXPECTED_UCR_CHILD_TABLE_PREFIX = 'tbl_80f005a0bdc2f0d0ff6f8293daee8f33_'
+EXPECTED_UCR_CHILD_TABLE_PREFIX = 'tbl_8e3a5ee0a4309ee52345c2cdcbb1215a_'
```

Comment:
This scares me a bit. The way our partitioning currently works is by inserting to these [tbl_md5sum tablenames](https://github.com/dimagi/architect/blob/b4605467bb74a68c996f4736ea749115ae8ba5c9/architect/databases/postgresql/partition.py#L25). I think we the table names change, they will [create a new trigger](https://github.com/dimagi/architect/blob/b4605467bb74a68c996f4736ea749115ae8ba5c9/architect/databases/postgresql/partition.py#L50) to insert into the new tablenames without changing the old tablenames or removing the old trigger.

I think this means that once the name is changed, it will be inserting the data twice in each table and there will potentially be two table names for all partitions

Reply:
Correct. That will only get impacted after the rename (phase 2). I'll work on dealing with that.

Reply:
878c0fb746a